### ISSUE TITLE
Poor performance on short files

### ISSUE THREAD

**Issue by aaronzira** [SeanNaren/deepspeech.pytorch#342]
Should it be expected that a model trained exclusively on 2-20s files would perform significantly worse on files at the lower end of that range and shorter?

In testing a model with:
- 7 gru layers
- 900 hidden units per layer
- adam
- swish activations
- iterative training using `--finetune` on a total of ~7500 hours (all in the [2, 20] second range)

on an isolated, [2, 20] second, in-house test set yields a best WER of **21.29** after gridsearching LM parameters, while a similar test set generated from the same original files, but with each clip in the [0.5, 5] second range yields a best WER of **27.88**. I can consistently observe this with different models and test sets.

Is there any way to avoid this, or is it to be expected given the nature of the model and training data? An immediate thought is to train specifically on files with shorter durations, but I'm wondering if there might be a more fundamentally sound approach.

**Comment by ryanleary** [comment]
Are you losing the latest code? Before Release v1.2 there was a bug in the WER calculation that basically returned the average WER per utterance instead of the overall WER. With utterances so short, the error rates tend to be higher because there are fewer words.. this effect is exacerbated by the WER calculation bug.

**Comment by aaronzira** [comment]
Ah yes, that was definitely it. Thanks!
